input/intro-notes/StructureDefinition-au-core-patient-notes.md

```diff
@@ -93,7 +93,7 @@
     
       1. GET [base]/Patient?_id=5678
 
-    *Implementation Notes:* Fetches a bundle with the requested Patient, instead of just the resource itself, and allows for the inclusion of additional search parameters such as _include, _revinclude, or _lastUpdated ([how to search by id of the resource](https://hl7.org/fhir/r4/search.html#id))
+    *Implementation Notes:* Fetches a bundle with the requested Patient, instead of just the resource itself, and allows for the inclusion of additional search parameters such as _include, _revinclude, or _lastUpdated ([how to search by id of the resource](https://hl7.org/fhir/r4/search.html#id)). Requester and responder have different conformance requirements for `_id`; requester **SHOULD** and responder **SHALL** support searching using the `_id` search parameter.
```

Comment:
Comment from @dt-r only visible via email notification:

Content with conformance verbs goes above the example. See for example point 3.
While there are multiple ways forwards likely the easiest way to do this is to Add another Implementation Note including "Implementation Note:" as a separate paragraph to state something like: Only for this search parameter are the conformance strengths different across requester and responder. Support for this search is optional for a requester.
Workshop this with @dbojicic

Reply:
Discussed with @dbojicic, have changed implementation note as suggested. 
Retained single 'Implementation Note**s**' rather than changing to 'Implementation Note' to retain consistency across IG.

Reply:
Text isn't right. I acknowledge that it's a copy paste of what I suggested to have text something like it. The language and structure of the phrasing isn't right. Re-workshop.